### DISCUSSION TITLE
tiptap fork ?

### DISCUSSION THREAD

**Discussion by ManUtopiK** [prosekit/prosekit#216] [General]
Prosekit looks very interesting ! Thanks for your work.

What is the main difference over tiptap editor ?
What are the benefits of prosekit ?

**Comment by ocavue** [comment]
I appreciate your kind words. Tiptap is a great project. ProseKit aims to offer better capabilities in the following two areas:

1. Focuses more on being expandable for complex editing projects. Its API is designed to make it easier for you to replace existing features.
2. Provides a headless component library that helps you add common editing features, like inline floating menu and slash menu.

**Reply by shyakadavis** [reply]
Hello, 👋 

Follow-up question on this, @ocavue 

In Tiptap, there appears to some sanitization issues, (https://github.com/ueberdosis/tiptap/issues/4007), and wanted to inquire if ProseKit does a better job in this regard.

Thanks.
